base/docker_healthcheck.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
Maybe to be a bit cleaner and just easier to see, what if we put all the files that are direct copies from upstream in a separate folder (i.e `upstream-scripts`) with a README pointing to their version and to give credit to the Jupyter team?

Reply:
But if all files would live there, maybe not needed......